recipes/_generate_password.rb

```diff
@@ -17,6 +17,7 @@
 
 execute 'change-admin-password' do
```

Comment:
To get the build going could we disable the failing rule on this line [like we did in the provider](https://github.com/cerner/cerner_splunk/blob/1.7.0/providers/forwarder_monitors.rb#L8)?